pkg/machinepool/machinepool.go

```diff
@@ -1447,8 +1454,13 @@ func getMachinePoolReplicas(cmd *cobra.Command,
 		})
 		if err != nil {
 			err = fmt.Errorf("Expected a valid number of replicas: %s", err)
+			return
```

Comment:
NP : Can we update this func to move away from named return values. It leads to errors like this that can be missed. 
Not a blocker for this PR, maybe we can follow up with that change

Reply:
Yeah I can follow up sometime soon with a refactor, i didn't like how it was set up like this, i definitely had a couple missed during testing haha